src/components/layout/Notification/RequestAuthoriyList.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from "classnames/bind";
+
+import { IRequestAuthority } from "@/types/dto";
+
+import styles from "./RequestAuthorityList.module.scss";
+import StatusChip from "./StatusChip";
+
+const cx = classNames.bind(styles);
+
+const RequestAuthorityList = ({ data }: { data: IRequestAuthority[] }) => {
+  if (data.length === 0) {
+    return (
+      <div className={cx("noContent")}>알림이 없습니다.</div>
+    );
+  }
+  return (
+    <div className={cx("container")}>
+      {data.map((item) => {
+        return (
+          <div className={cx("content")} key={item.requestAuthorityId}>
+            <div className={cx("sender")}>
+              <span className={cx("senderNickname")}>{item.sender.nickname}</span>
```

Comment:
저희가 노션에 nickname(loginId)로 하기로 해서 이렇게 바꿔야될 것 같아요!

<img width="290" alt="스크린샷 2023-10-26 오전 10 58 04" src="https://github.com/Bluekey-Payment-System/BPS-FE/assets/103362820/b495542c-865c-4285-b9d9-293aa66e2c3e">

```suggestion
              <span className={cx("senderNickname")}>{item.sender.nickname(item.sender.loginId)}</span>
```

Reply:
아아! 맞다! 어휴 중요한걸 놓칠 뻔했네요 감사합니다!